src/core/styles/fonts.css

```
@@ -0,0 +1,15 @@
+@font-face {
+  font-family: "Inter";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)